connexta/alliance
# CAL-29 MPEG-TS/KLV Input Transformer

#### What does this PR do?

Add MPEGTS input transformer that extracts KLV metadata from the transport stream and adds it the metacard.

#### Who is reviewing it?
@rzwiefel @jrnorth @bdeining  @jlcsmith @beyelerb 

#### How should this be tested?
1) Use a DDF with the new metacard validation (latest 2.9.0-SNAPSHOT)
2) Build alliance-mpegts-transformer and alliance-video-app
3) copy alliance-video-app-0.1-SNAPSHOT.kar into ddf-2.9.0-SNAPSHOT/deploy
4) ingest FFMPEG sample data https://samples.ffmpeg.org/MPEG2/mpegts-klv/Night%20Flight%20IR.mpg (but rename file to .ts extension)
5) search for video file, confirm that the content type is video/mp2t and there is a metadata field called "Image Source Sensor". The value should be "IR". 

#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:
